### ISSUE TITLE
"This is a test" checkmark needs new image

### ISSUE THREAD

**Issue by ianb** [mozilla/firefox-voice#1808]
"This is a test" gives this:

<img width="402" alt="image" src="https://user-images.githubusercontent.com/44390/87330838-bffd3080-c4fe-11ea-8c66-2e7bc5d5f159.png">

The background of the checkmark isn't great.

Also the popup should probably stay open slightly longer than it does currently.

**Comment by awallin** [comment]
Can we use the existing checkmark image that's typically seen when an intent is completed?

**Comment by ianb** [comment]
Yes, I think we could put that in, that would require using the Zap component instead of an image.
